### ISSUE TITLE
MetaMask Firefox extension Transactions still pending

### ISSUE THREAD

**Issue by BigT2085** [MetaMask/faq#105]
I have just installed the Metamask Extention in Firefox, this is causing firefox to be extremely laggy, I would uninstall it to use with chrome however there are still some transactions pending, seems to be due to  gas  but i have bumped the gas limit to over 120 GWEI and it is still not working. Is there any workarounds for this? I bought an ERC20 token from the ethercraft shop, item has now doubled in price so i would prefer to not cancel the transaction. any help would be appreciated. Thanks

**Comment by frankiebee** [comment]
@BigT2085 do you have any transactions pending before this one?

**Comment by BigT2085** [comment]
yeah there are a couple of transactions there, but i cant see them as you cant scroll down in the browser.

**Comment by danfinlay** [comment]
Later today we should have published our fix to the scrolling issue.

**Comment by DVS1701** [comment]
How do I view my balance in latium and withdraw some ETH from my ETHSCAN Acct.
